packages/components/src/popover/test/index.js

```diff
@@ -15,29 +15,6 @@ describe( 'Popover', () => {
 		}
 	} );
 
-	it( 'should focus when opening in response to keyboard event', () => {
```

Comment:
The ref callbacks are not called making this very hard to test, I spent some time on it and didn't find how to trigger these in the tests. Maybe someone has an idea here. Otherwise, this use-case is already tested in several places though in e2e tests.

Reply:
I agree e2e are much better for these